src/assets/scss/components/_body.scss

```diff
@@ -276,106 +270,113 @@
                 @include textHover($black-color)
             }
         }
+
         font-size: 4em;
         font-weight: 400;
         text-transform: uppercase;
     }
 
-    .project-content{
+    .project-content {
         text-align: justify;
         padding-top: 20px;
         font-size: 2em;
         color: $text-color;
         font-weight: 300;
-        
+
     }
-    .btn-seemore{
+
+    .btn-seemore {
         position: absolute;
         bottom: 0;
         right: 0;
+    }
 }
-}
+
 .project-outstanding {
-   
-    .project-outstanding_wrapper_separation{
 
-    }
-    .project-outstanding_wrapper-title{
+    .project-outstanding_wrapper_separation {}
+
+    .project-outstanding_wrapper-title {
         color: $white-color;
         position: relative;
         font-size: 3em;
         text-align: center;
         padding-bottom: 20px;
         font-weight: 600;
-        >span{
+
+        >span {
             color: $primari-color;
         }
-        }
-    
+    }
+
 
     .project-outstanding_inner {
-        padding: 20px;
+        padding-bottom: 80px;
 
-            .project-outstanding_item {
-                padding-top: 70%;
-            }
+        .project-outstanding_item {
+            padding-top: 70%;
+        }
 
-            .project-outstanding_content {
-                background-color: rgba($color: #3b3939, $alpha: 1.0) ;
-                min-height: 250px;
-                .project-outstanding_title {
-                    >span {
-                        text-transform: uppercase;
-                        font-size: 1.6em;
-                        color: $primari-color;
-                    }
+        .project-outstanding_content {
+            background-color: #26222285;
+            min-height: 250px;
 
-                    color: white;
+            .project-outstanding_title {
+                >span {
+                    text-transform: uppercase;
                     font-size: 1.6em;
-                    text-align: center;
-                    padding: 10px;
-                    cursor: pointer;
-                    &:hover{
-                        >span{
-                            @include textHover($white-color)
-                        }
-                    }
-
+                    color: $primari-color;
                 }
 
-                .project-outstanding_text {
-                    color: $white-color;
-                    font-size: 1.6em;
+                color: white;
+                font-size: 1.6em;
+                text-align: center;
+                padding: 10px;
+                cursor: pointer;
 
+                &:hover {
+                    >span {
+                        @include textHover($white-color)
+                    }
                 }
 
+            }
+
+            .project-outstanding_text {
+                color: $white-color;
+                font-size: 1.6em;
+                padding: 0 20px;
 
             }
 
-        
+
+        }
+
```

Comment:
@vihan85 
Tại sao lại cách ra nhiều dòng như vậy ?

Reply:
Hi @huulochuynh
This issue has been resolved

Thanks best & regards